src/node.cc

```diff
@@ -2456,44 +2456,46 @@ static void GetParentProcessId(Local<Name> property,
 
 
 static Local<Object> GetFeatures(Environment* env) {
-  EscapableHandleScope scope(env->isolate());
+  Isolate* isolate = env->isolate();
+  EscapableHandleScope scope(isolate);
+  Local<Boolean> true_value = True(isolate);
```

Comment:
Why do we need these two separate variables instead of using `True(isolate)` and `False(isolate)` inline like before?